src/styles/_catalog.scss

```
@@ -1,10 +1,21 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
-
+  display: grid;
+  grid-template-columns: repeat(1, $card-width);
```

you don't need to use `repeat()` function here, when we are repeating our column 1 time - it's anyway 1 column
```suggestion
  grid-template-columns: $card-width;
```